app/javascript/elm/src/Main.elm

```diff
@@ -403,32 +407,36 @@ update msg model =
             in
             ( { model | overlay = Overlay.update overlay model.overlay }, Cmd.none )
 
-        ToggleIndoor ->
+        ToggleIndoor newValue ->
             let
                 ( subModel, subCmd ) =
                     deselectSession model
             in
-            if subModel.isIndoor then
-                ( { subModel | isIndoor = False, overlay = Overlay.update (RemoveOverlay IndoorOverlay) model.overlay }
-                , Cmd.batch [ Ports.toggleIndoor False, subCmd ]
-                )
+            if subModel.isIndoor == newValue then
+                ( subModel, Cmd.none )
```

Comment:
If you are using submodel, then execute the subCmd as well, together they are responsible for deselecting a session. Right now there is a bug:
- go to fix active
- select a session
- click active again
- looks like session is deselected
- refresh the page
- it's still selected

Reply:
fixed so that nothing is happening